.github/workflows/lint.yml

```diff
@@ -0,0 +1,26 @@
+name: Lint check
+on:
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  lint-check:
+    name: Lint check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the Repository
+        uses: actions/checkout@v4
+
```

Comment:
shouldn't we setup nodejs here?

Reply:
I think its not required as ubuntu-latest comes with yarn and node